packages/firebase_database/firebase_database/example/lib/main.dart

```diff
@@ -61,12 +49,14 @@ class _MyHomePageState extends State<MyHomePage> {
     final FirebaseDatabase database = FirebaseDatabase(app: widget.app);
     _messagesRef = database.reference().child('messages');
     database.reference().child('counter').once().then((DataSnapshot snapshot) {
+      print(snapshot.value);
```

Comment:
Should probably remove.

Reply:
I'm curious why this request was closed as resolved with no discussion